third_party/eigen3/unsupported/Eigen/CXX11/src/FixedPoint/PacketMathAVX512.h

```diff
@@ -0,0 +1,545 @@
+#ifndef THIRD_PARTY_EIGEN3_UNSUPPORTED_EIGEN_CXX11_SRC_FIXEDPOINT_PACKETMATHAVX512_H_
```

Comment:
It worries me that (AFAICT) there are no direct unit tests in the open source version of TensorFlow for the functions in third_party/eigen3.